scripts/cangen/cangen/can_generator.py

```diff
@@ -22,43 +24,47 @@
 MSG_REGISTRY_FILE_NAME = "_msg_registry.h"
 CAN_MESSAGES_FILE_NAME = "_can_messages.h"
 
+OUTPUT_DIR = "generated/can"
```

Comment:
This should be moved to config.py and used as a default argument to config.pop('outputPath')